src/h5web/visualizer/Visualizer.module.css

```diff
@@ -27,7 +27,7 @@
 
 .error {
   padding: 1em;
-  color: var(--danger);
+  color: brown;
```

Comment:
Should it be `var(--h5web-error--color, brown)` ?

Reply:
Well, `Visualizer` is not exposed to the lib, so there's no point in making it configurable. Since `--h5web-error--color` doesn't have a value in H5Web (because the default value works for us), then there's no point in using the `var(--h5web-error--color, brown)` syntax.